### DISCUSSION TITLE
[Weekly Curation Report]: <May 13> - <June 20>

### DISCUSSION THREAD

**Discussion by IsabelWassink** [wikipathways/wikipathways-help#40] [curation-reports]
### Recent changes

Recent Changes: 
The last curation report was from May 12 so I started from there. There were a lot of recent changes but they all looked okay. 

### Under construction

Under Construction: 
Most pathways under construction are still actively edited. 
- [WP5342](https://classic.wikipathways.org/index.php/Pathway:WP5342): last edit was in March but is tagged as 'test pathway', so I am not sure how to proceed
- [WP5361](https://classic.wikipathways.org/index.php/Pathway:WP5361): is not actively edited but I am not sure if it should be approved or if it still needs work

### New pathways

New pathways: 
There are 29 new pathways.
- Three are part of the WikiPathways Academy
- I approved [WP5390](https://classic.wikipathways.org/index.php/Pathway:WP5390) and [WP5389](https://classic.wikipathways.org/index.php/Pathway:WP5389) (but not sure which ontology terms to add to this one)
- Still under construction: [WP5387](https://classic.wikipathways.org/index.php/Pathway:WP5387), [WP5386](https://classic.wikipathways.org/index.php/Pathway:WP5386), [WP5385](https://classic.wikipathways.org/index.php/Pathway:WP5385), [WP5381](https://classic.wikipathways.org/index.php/Pathway:WP5381), [WP5380](https://classic.wikipathways.org/index.php/Pathway:WP5380), [WP5376](https://classic.wikipathways.org/index.php/Pathway:WP5376), [WP5373](https://classic.wikipathways.org/index.php/Pathway:WP5373), [WP5372](https://classic.wikipathways.org/index.php/Pathway:WP5372), [WP5368](https://classic.wikipathways.org/index.php/Pathway:WP5368), [WP5367](https://classic.wikipathways.org/index.php/Pathway:WP5367), [WP5366](https://classic.wikipathways.org/index.php/Pathway:WP5366), and [WP5362](https://classic.wikipathways.org/index.php/Pathway:WP5362)
- Multiple of the new pathways are duplicates, see Deleted Pathways

### Deleted Pathways

Deleted pathways: 
- [WP5384](https://classic.wikipathways.org/index.php/Pathway:WP5384), [WP5383](https://classic.wikipathways.org/index.php/Pathway:WP5383) and [WP5388](https://classic.wikipathways.org/index.php/Pathway:WP5388) are all duplicates of [WP5385](https://classic.wikipathways.org/index.php/Pathway:WP5385) and should be deleted, but I don't have deletion rights
- [WP5387](https://classic.wikipathways.org/index.php/Pathway:WP5387), [WP5382](https://classic.wikipathways.org/index.php/Pathway:WP5382), [WP5375](https://classic.wikipathways.org/index.php/Pathway:WP5375) and [WP5374](https://classic.wikipathways.org/index.php/Pathway:WP5374) also seem to be duplicates 
- [WP5371](https://classic.wikipathways.org/index.php/Pathway:WP5371), [WP5370](https://classic.wikipathways.org/index.php/Pathway:WP5370) and [WP5369](https://classic.wikipathways.org/index.php/Pathway:WP5369) are all duplicates of each other and were already tagged for deletion, but maybe one of them should stay?
- [WP5364](https://classic.wikipathways.org/index.php/Pathway:WP5364) is a duplicate of [WP5365](https://classic.wikipathways.org/index.php/Pathway:WP5365) and should be deleted 
- [WP5363](https://classic.wikipathways.org/index.php/Pathway:WP5363) is also a duplicate and should be deleted

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by danidi** [comment]
Sorry for the duplicates! My students had some troubles with PathVisio giving an error, but nevertheless uploading the pathway without them noticing it. I tried to tag all duplicates for deletion, but kept them for now just in case. 
[WP5382](https://classic.wikipathways.org/index.php/Pathway:WP5382) is to be kept, I tagged all others as duplicate upload for deletion (one just now).
[WP5371](https://classic.wikipathways.org/index.php/Pathway:WP5371), [WP5370](https://classic.wikipathways.org/index.php/Pathway:WP5370) and [WP5369](https://classic.wikipathways.org/index.php/Pathway:WP5369) are all duplicates of [WP5372](https://classic.wikipathways.org/index.php/Pathway:WP5372), which was renamed afterwards.

**Comment by khanspers** [comment]
Thanks @IsabelWassink! 

- [WP5342](https://classic.wikipathways.org/index.php/Pathway:WP5342): The content is interesting and not an obvious test pathway, so I contacted the original author. And I also made some minor updates to fix labels, interactions, etc.
- [WP5361](https://classic.wikipathways.org/index.php/Pathway:WP5361): belongs to @marvinm2, maybe its ready?
- Deleted: WP5388, WP5374, WP5375, WP5387, WP5371, WP5369, WP5370, WP5364, WP5363
- WP5372 wasnt tagged with anything so I added "Under Construction". @danidi  if its ready, please tag as "Approved".

**Comment by marvinm2** [comment]
WP5361 (and also WP5389) are indeed both AOP pathways and require some additional work. 
WP5361 is a comparison between the COVID AOP developed in CIAO project and inflammatory AOP under development in the project imSAVAR
WP5389 is and AOP related to the case study of nephrotox in VHP4Safety, but this one is getting updated
